CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ find_package(OpenSSL REQUIRED)
 find_package(uni-algo QUIET)
 find_package(dice-hash QUIET)
 find_package(dice-sparse-map QUIET)
+find_package(unordered_dense QUIET)
```

Comment:
Motivation for including unordered_dense: The number of prefixes is typically low, but their usage might occur often => faster map better

But I actually have no idea if this makes any appreciable difference because we don't really have datasets in turtle format

Reply:
I would only switch to unordered_dense when we see that it is a bottle neck. For the typically low amount of prefixes (<20 from my experience) I think boost's flat_map should be just as fast. 

For now, I prefer to keep our dependencies short.